### ISSUE TITLE
300W data set   There is a problem with the test set and training set on the 300W data set

### ISSUE THREAD

**Issue by iam-zhanghongliang** [polarisZhao/PFLD-pytorch#49]
The training set does not contain IBUG, and IBUG belongs to the validation set.
![image](https://user-images.githubusercontent.com/45345828/103512567-e98e8780-4ea3-11eb-82e7-b6158593cab6.png)
